docker/Dockerfile

```diff
@@ -12,7 +12,15 @@ RUN apt-get install -y --no-install-recommends \
 # Add curl, bzip2
 RUN apt-get update -qq && apt-get -y --no-install-recommends install \
     bzip2 \
-    curl 
+    curl
+
+# Install pip3 and installation tools
+RUN apt-get -y --no-install-recommends install \
+    python3-pip  python3-dev
```

Comment:
Do you want to specify python version?

Reply:
Probably should for freezing purposes. :+1: